packages/wrangler/templates/middleware/middleware-ensure-req-body-drained.ts

```diff
@@ -0,0 +1,14 @@
+import type { Middleware } from "./common";
+
+const drainBody: Middleware = async (request, env, _ctx, middlewareCtx) => {
+	try {
+		return await middlewareCtx.next(request, env);
+	} finally {
+		if (!request.bodyUsed) {
+			_ctx.waitUntil(request.arrayBuffer());
```

Comment:
```suggestion
			ctx.waitUntil(request.arrayBuffer());
```

Reply:
I wonder if `request.body.cancel()` would work better here?

Reply:
Interesting!!

Reply:
Nope. That didn't help. Still fails the test.